src/Pages/SolvingQuestion.tsx

```diff
@@ -58,6 +68,17 @@ export function SolvingQuestion() {
         else console.log("WRONG")
     }
 
+    function submitReport() {
```

Comment:
use `useCallback` to define functions inside components. If you don't use a hook and use the function in other hooks (e.g., useEffect), the hook will be triggered every time components are rendered, which is not desired in most cases.
```suggestion
   const submitReport = useCallback(() => {
   ...
   }, [])
```

Reply:
understand. I will fix it